lib/phoney/contacts.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Phoney.Contacts do
+  use Ash.Domain
+
+  resources do
+    resource Phoney.Contacts.Contact
+  end
+
+  def read!(query \\ GistAsh.Gists.Gist, opts \\ []) do
+    query
+    |> Ash.Query.for_read(:read)
+    |> Ash.read!(domain: __MODULE__, actor: Keyword.get(opts, :actor))
+  end
```

Comment:
Remove that and replace with code_interface: https://hexdocs.pm/ash/code-interfaces.html